src/pages/components/ProductSummary.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react'
+import { Box, Composition } from 'atomic-layout'
+import styled from 'styled-components'
+import { Image } from '../../atoms/Image'
+import { Heading } from '../../atoms/Heading'
+import { Button } from '../../atoms/Button'
+import { FaFacebookF, FaTwitter, FaInstagram } from 'react-icons/fa'
+import { Rating } from '../../molecules/Rating'
+
+const templateMobile = `
+	gallery
+	summary
```

Comment:
These strings use 4 spaces of indentation. Maybe use 2 as you do everywhere else?

```diff
-    gallery
+  gallery
```

Reply:
"Works fine on my machine" :D